link/src/test/java/com/stripe/android/link/model/StripeIntentKtxTest.kt

```diff
@@ -0,0 +1,67 @@
+package com.stripe.android.link.model
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.link.TestFactory
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class StripeIntentKtxTest {
+
+    @Test
+    fun `When test mode and test account then all funding sources are enabled`() {
+        val supportedTypes = stripeIntent(liveMode = false)
+            .supportedPaymentMethodTypes(linkAccount("test+multiple_funding_sources@test.abc"))
+        assertThat(supportedTypes).containsExactly("card", "bank_account")
+    }
+
+    @Test
+    fun `When live mode and test account then uses intent funding sources`() {
+        val supportedTypes = stripeIntent(listOf("card"))
```

Comment:
nit: using a named parameter here (ie writing this as `stripeIntent(fundingSources = listOf("card"))`) would be useful, because there are multiple lists of PMs that could be specified on the intent and that would make it more clear which was being used

Reply:
Just realized I missed this. It should be fixed now